pandas/plotting/_matplotlib/core.py

```diff
@@ -102,13 +104,15 @@ def __init__(
         table=False,
         layout=None,
         include_bool=False,
+        column=None,
         **kwds,
     ):
 
         import matplotlib.pyplot as plt
 
         self.data = data
         self.by = by
+        self.column = [column] if isinstance(column, str) else column
```

Comment:
only reason i did this was the accepted column type is str or sequence, in case it is a tuple. then `column if isinstance(column, list) else [column]` this would give an incorrect value for column. any suggestions?

Reply:
Should do `if not isinstance(column, list):` as the condition and then `column = [column]` if true

Reply:
changed! thx